tests/integration/ceph_multi_cluster_test.go

```diff
@@ -92,6 +92,7 @@ func (s *MultiClusterDeploySuite) SetupSuite() {
 		Namespace:         "multi-external",
 		OperatorNamespace: s.settings.OperatorNamespace,
 		RookVersion:       s.settings.RookVersion,
+		CephVersion:       installer.PacificVersion,
```

Comment:
only adding this Ceph version should also work right? since we'll have the same ceph version to pull for toolbox pod.

Reply:
Yes, I expect this to help with the multicluster test, so the rest of the refactor is just a precaution for other tests.